src/hermes_claude_agent_sdk/parity/assembler.py

```
@@ -0,0 +1,448 @@
+            "pack_counts": dict(self.pack_counts), "accounting": dict(self.accounting),
+            "source_keys": [{"pack_id": p, "row_id": r} for p, r in self.source_keys], "rows": list(self.rows), "source_rows": list(self.rows),
+            "residual_gaps": list(self.residual_gaps), "gaps": list(self.residual_gaps),
+            "gap_counts": dict(sorted(gap_counts.items())),
+            "diagnostics": [item.to_dict() for item in self.diagnostics],
+            "missing_session_scope": [{"pack_id": p, "row_id": r} for p, r in self.missing_session_scope], "missing_session_scope_assignments": [{"pack_id": p, "row_id": r} for p, r in self.missing_session_scope], "missing_session_scope_count": len(self.missing_session_scope),
+            "missing_sdk_proof_kind": [{"pack_id": p, "row_id": r} for p, r in self.missing_sdk_proof_kind], "missing_sdk_primary_proof_kinds": [{"pack_id": p, "row_id": r} for p, r in self.missing_sdk_proof_kind], "missing_sdk_proof_kind_count": len(self.missing_sdk_proof_kind),
+            "sdk_rows_without_required_path": [{"pack_id": p, "row_id": r} for p, r in self.sdk_rows_without_required_path], "sdk_no_required_path_rows": [{"pack_id": p, "row_id": r} for p, r in self.sdk_rows_without_required_path], "sdk_rows_without_required_path_count": len(self.sdk_rows_without_required_path),
+            "inventory_gap_rows": [{"pack_id": p, "row_id": r} for p, r in self.inventory_gap_rows], "inventory_status": "PENDING" if self.inventory_gap_rows else "UNRESOLVED", "issue_16": dict(self.issue_16), "sdk_stop": dict(self.issue_16), "provenance": dict(self.provenance), "catalog": None, "catalog_sha256": None,
+        }
+
+    def __getitem__(self, key: str) -> Any:
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `__getitem__`

(ANN401)